src/controllers/users.controller.ts

```diff
@@ -60,6 +60,18 @@ class UsersController {
       next(error);
     }
   };
+
+  public manageGroups = async (req: Request, res: Response, next: NextFunction) => {
+    try {
+      const data = req.body;
```

Comment:
to be consistent with the rest of the code here should be groupId instead of data or we can create a new DTO object with the groupId as a property  

that groupId should go as a parameter to the userService.manageGroups instead of data